.travis.yml

```diff
@@ -6,8 +6,12 @@ env:
       - UNICODE_WIDTH=32
       - NP_BUILD_DEP="numpy==1.8.2"
       - NP_TEST_DEP="numpy==1.13.1"
+      - CYTHON_BUILD_DEP="cython==0.28.5"
```

Comment:
the 0.28.5 dependency is only there for python3.5 and conda. For the nightly builds and for python>=3.6 I think we can keep it as cython>=0.29.

Reply:
Since our release builds uses `cython==0.28.5`, our nightly builds should stick to this. This keeps the nightly builds as close as possible to the release builds.